src/cls/IPM/Storage/Module.cls

```diff
@@ -1038,7 +1040,7 @@ Method GetResolvedReferences(Output pReferenceArray, pLockedDependencies As %Boo
 				}
 				
 				// Don't pass along the phase
-				Set pReferenceArray(tModName_".ZPM") = ..Name
+				Set pReferenceArray(tModName, tModName_".ZPM") = ..Name
```

Comment:
For same reason as above, can remove this.

Reply:
So the above one is needed but this one isn't since it calls GetResolvedReferences() and not ResolveChildren()